### PR TITLE
Add outline transition to navigate / edit mode switching

### PR DESCRIPTION
**Pull request by mtias**
This attempts to make switching between "Edit" and "Navigate" perceptually more fluid. It also removes the background fade on paragraph blocks when navigate mode is engaged.

cc @jasmussen

### CODE REVIEW

File: packages/block-editor/src/components/block-list/style.scss [hunk 1]
```diff
@@ -198,6 +199,18 @@
...
 
+	&::after {
+		content: "";
+		position: absolute;
+		top: 0;
+		right: 0;
+		bottom: 0;
+		left: 0;
+		border-radius: $radius-block-ui;
+		transition: box-shadow 0.1s ease-in;
```
Comment (jasmussen) [inline packages/block-editor/src/components/block-list/style.scss]: Can you include `@include reduce-motion("transition");` after this one? That should be sufficient to disable it for people who choose to.

**Review by jasmussen — APPROVED** [review]
This is what I see:

fade

This is working well for me as a nice little refinement.

File: packages/block-editor/src/components/block-list/style.scss (hunk 1 above)
Comment (mtias) [reply packages/block-editor/src/components/block-list/style.scss]: Yes, good call. There's also probably some redundancy on the initial styles declaration now that the default ::after includes it.

**Comment by mtias** [comment]
The gif barely captures it :)

**Comment by jasmussen** [comment]
It doesn't. Too low framerate! But I _saw_ it. Trust me.

**Comment by jasmussen** [comment]
I think this one can be closed :) — the effect has been implemented by Q!

**Pull request closed without merging**